src/index.js

```diff
@@ -0,0 +1,69 @@
+import { createProject, createTodo } from './create';
+import {
+  rendertask, renderProjects, projects, display, save, LOCAL_STORAGE_SELECTED_PROJECT,
+} from './project';
+
+const projectFormBody = document.getElementById('project_form_body');
+const taskFormBody = document.getElementById('task_form_body');
+
+const newProjectInput = document.querySelector('[data-new-project-input]');
+
+const newTodoInputTitle = document.querySelector('[data-new-todo-title-input]');
+const newTodoInputDesc = document.querySelector('[data-new-todo-desc-input]');
+const newTodoInputPrior = document.querySelector('[data-new-todo-prior-input]');
+const newTodoInputDate = document.querySelector('[data-new-todo-date-input]');
+const newTodoInputTime = document.querySelector('[data-new-todo-time-input]');
+const newTodoInputNote = document.querySelector('[data-new-todo-note-input]');
+
+const addProject = document.getElementById('add_project');
+const addTask = document.getElementById('add_task');
+
+const projectForm = document.getElementById('project_form');
+const taskForm = document.getElementById('task_form');
+
+addProject.addEventListener('click', () => {
+  projectFormBody.style.visibility = 'visible';
```

Comment:
- [ ] [REQUIRED] 
Keep all the styling logic in CSS files (If you need to change something like a background image, add the classes with the different backgrounds to the CSS and then change the class name attribute with JavaScript) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/styling_requirement.png) from Pathwright)